### PR TITLE
auto submitting sitemap to google and bing

### PR DESCRIPTION
**Pull request by eishay**
@andrewconner @raymondng

### CODE REVIEW

**Comment by andrewconner** [comment]
Awesome

File: kifi-backend/modules/shoebox/app/com/keepit/common/seo/SiteMapGenerator.scala [hunk 1]
```diff
@@ -33,30 +34,30 @@ class SiteMapGeneratorPluginImpl @Inject() (
   override def enabled: Boolean = true
   override def onStart() {
     for (app <- Play.maybeApplication) {
-      val (initDelay, freq) = if (Play.isDev) (30 seconds, 10 minutes) else (15 minutes, 60 minutes)
+      val (initDelay, freq) = (15 minutes, 60 minutes)
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/common/seo/SiteMapGenerator.scala]: Mind setting the `15 minutes` to higher, like `60 minutes`? As-is, server restarts within an hour will call it many times.

**Comment by raymondng** [comment]
Nice :+1:

File: kifi-backend/modules/shoebox/app/com/keepit/common/seo/SiteMapGenerator.scala (hunk 1 above)
Comment (eishay) [inline kifi-backend/modules/shoebox/app/com/keepit/common/seo/SiteMapGenerator.scala]: sure

**Pull request merged**